### PR TITLE
Bundled package

### PR DESCRIPTION
**Pull request by displague**
### Description of your changes
In order to fix image publishing, we'll need provider-jet-equinix image repos created (per #5). This PR sets up the new single image repo convention.

Publishes a single image for package metadata and the controller.
Uploads the image to the Upbound registry.

Borrows from https://github.com/crossplane-contrib/provider-gcp/pull/456

Part of #5 

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,14 +47,29 @@ GO111MODULE = on
 # ====================================================================================
 # Setup Kubernetes tools
 
+UP_VERSION = v0.13.0
```
Comment (displague) [inline Makefile:50]: I'm not sure what this is for

File: Makefile [hunk 2]
```diff
@@ -47,14 +47,29 @@ GO111MODULE = on
...
 -include build/makelib/k8s_tools.mk
 
 # ====================================================================================
 # Setup Images
 
-DOCKER_REGISTRY ?= crossplane
-IMAGES = provider-jet-equinix provider-jet-equinix-controller
--include build/makelib/image.mk
+IMAGES = provider-equinix
+-include build/makelib/imagelight.mk
```
Comment (displague) [inline Makefile:58]: The build/ directory in this project is a copy of the relevant files (pattern taken from the provider-jet-template) from https://github.com/upbound/build/commit/e018ec6.

Comment (displague) [reply Makefile:58]: We may need to update build files if this is not sufficient.

**Comment by displague** [comment]
I believe we'll need the Docker crossplane-contrib provider-jet-equinix image repository created in advance of a merge so the 'latest' image can be pushed.

I've created https://cloud.upbound.io/equinix/repositories/provider-jet-equinix but we'll need a valid secret added to the GH settings.

File: Makefile (hunk 1 above)
Comment (hasheddan) [reply Makefile:50]: Sets the version for https://github.com/upbound/up 👍🏻

File: Makefile (hunk 2 above)
Comment (hasheddan) [reply Makefile:58]: I _think_ this should be sufficient, but updating to latest may be required for Go to work in CI after merge

File: Makefile [hunk 3]
```diff
@@ -47,14 +47,29 @@ GO111MODULE = on
...
-DOCKER_REGISTRY ?= crossplane
-IMAGES = provider-jet-equinix provider-jet-equinix-controller
--include build/makelib/image.mk
+IMAGES = provider-jet-equinix
+-include build/makelib/imagelight.mk
+
+# ====================================================================================
+# Setup XPKG
+
+XPKG_REG_ORGS ?= xpkg.upbound.io/crossplane index.docker.io/crossplane
```
Comment (hasheddan) [inline Makefile]: If you want to publish in equinix org on marketplace we would need to update to `xpkg.upbound.io/equinix` -- is that preferable?

**Review by displague — COMMENTED** [review]
...

**Review by displague — COMMENTED** [review]
...

**Review by hasheddan — APPROVED** [review]
Thanks @displague!

**Pull request merged**